index.bs

```diff
@@ -1660,7 +1660,7 @@ session.ProxyConfiguration = {
    session.DirectProxyConfiguration //
    session.ManualProxyConfiguration //
    session.PacProxyConfiguration //
-   session.SystemProxyConfiguration //
+   session.SystemProxyConfiguration
```

Comment:
FWIW, trailing `//` are also valid per the CDDL grammar (and I don't see any text in the RFC that forbidd them), so if you keep trailing commas, you could keep `//` for the same reason.